wagtailmedia/widgets.py

```diff
@@ -19,6 +29,19 @@ def __init__(self, **kwargs):
         super(AdminMediaChooser, self).__init__(**kwargs)
         self.media_model = get_media_model()
 
+    def get_value_data(self, value):
+        if not WAGTAIL_VERSION > (2, 13):
```

Comment:
If I just return json regardless of wagtail version, I get this error message from wagtail 2.12.6:
![json_traceback](https://user-images.githubusercontent.com/499500/126960437-e267bd4e-914b-4aab-84ba-e7023ccac14a.png)

Reply:
🤔 have you tested this with the updated `render_html`? seems to work fine for me locally

This is what the image chooser widget does: https://github.com/wagtail/wagtail/blob/571b9e1918861bcbe64a9005a4dc65f3a1fe7a15/wagtail/images/widgets.py#L26-L45

Reply:
Ups, fixed with last commit :).

Reply:
> 🤔 have you tested this with the updated `render_html`? seems to work fine for me locally

Yes sorry, didn't read far enough at first. Worked after `render_html`modifications.

> This is what the image chooser widget does: https://github.com/wagtail/wagtail/blob/571b9e1918861bcbe64a9005a4dc65f3a1fe7a15/wagtail/images/widgets.py#L26-L45

Hmm, this seems a little bit odd to me:

```python
value_data = value_data or {}
```

It's probably ok for wagtail itself as long as it's guaranteed that get_value_data is called which returns either a dict or None. But if value_data is the model id this will fail for  primary keys that evaluate to False (like `0`- unlikely, but possible).